rtl/include/eclhelper_base.hpp

```diff
@@ -1816,9 +1816,8 @@ class CThorJoinArg : implements IHThorJoinArg, public CThorArg
     virtual size32_t onFailTransform(ARowBuilder & rowBuilder, const void * _left, const void * _right, IException * e) { return 0; }
 
 //Join:
-    virtual size32_t transform(ARowBuilder & rowBuilder, const void * _left, const void * _right) { return 0; }
```

Comment:
@ghalliday FYI as far as I can tell these functions were made common with the denormalize transform below and were never called anymore... but please verify.